rpc/README.md

```diff
@@ -2001,6 +2002,41 @@ http://localhost:8114
 }
 ```
 
+### `sync_state`
+
+Returns sync state of this client
+
+
+#### Examples
+
+```bash
+echo '{
+    "id": 2,
+    "jsonrpc": "2.0",
+    "method": "sync_state",
+    "params": []
+}' \
+| tr -d '\n' \
+| curl -H 'content-type: application/json' -d @- \
+http://localhost:8114
+```
+
+```json
+{
+    "id": 2,
+    "jsonrpc": "2.0",
+    "result": {
+        "best_known_block_number": "0x248623",
```

Comment:
I think the best known block timestamp is also useful. The client can use the timestamp to check whether the node has synchronized to the recent tip header.

Reply:
Another field suggestion: inflight_peers_count

Reply:
> Another field suggestion: inflight_peers_count

I don’t think it’s useful, unless you can display all the synced peer information